### DISCUSSION TITLE
Strange behaviour with navigation on Android using decompose and jetbrains complose

### DISCUSSION THREAD

**Discussion by serhiynovos** [arkivanov/Decompose#278] [Q&A]
Hi. Can you please check this issue https://github.com/JetBrains/compose-jb/issues/2505 . Maybe you have any ideas where can be problem on Decompose or compose side. On desktop version everything is working fine with no problems. 

One thing: now I'm using material 3, can it also cause this issues as it's in experimental state ?

**Comment by arkivanov** [comment]
This looks like a know issue, right? https://issuetracker.google.com/issues/258907850

**Reply by serhiynovos** [reply]
Yes. I got the same stack trace. It happens on navigation for me. Tried 1.2.1 and 1.3.0 alpha, the same crashes

**Reply by arkivanov** [reply]
Yeah, thanks for raising. Looks like nothing to do from Decompose side.

**Reply by arkivanov** [reply]
You can put +1 in the ticket.

**Reply by serhiynovos** [reply]
But it's sad it's happening to compose in stable versions (

**Reply by arkivanov** [reply]
Maybe try 1.3.1? It was reported there is no crash on 1.3.0-beta03.

**Reply by serhiynovos** [reply]
Do you mean use separately jetbrains compose for desktop and jetpack compose for Android ?

**Reply by serhiynovos** [reply]
I'll check later with jetbrains compose 1.3.0 beta 03, currently my laptop died due to no electricity because of missile attack

**Reply by serhiynovos** [reply]
@arkivanov Thanks for your quick response

**Reply by arkivanov** [reply]
Yeah, you can try `1.3.0-beta03`, it should be based on `1.3.1` for Android AFAIK.

**Reply by serhiynovos** [reply]
Tried 1.3.0-beta03 jetbrains compose, problem is not fixed yet

**Reply by arkivanov** [reply]
So you have a reproducer?
